packages/overreact/src/spec/response-handler/refetch.js

```diff
@@ -59,6 +59,11 @@ export default function handler(environment, processedResponse, request) {
       store.getRecordGroup(context.schemaNode.modelName)
         .addOrUpdateRecords(dataWithId, request);
 
+      // when the return value is empty, we still need to notify the hook who trigger this call
+      if (dataWithId.length === 0) {

```

Comment:
Let's have a test first.

Reply:
yeah, no test here in git will make it prone to regression, we may need to enable it soon.